tests/main/install-snaps/task.yaml

```diff
@@ -0,0 +1,82 @@
+summary: Check install popular snaps
```

Comment:
I'm not super keen on having this in the main suite where it blocks all PRs.

Ideally (at least for me, obviously) this would be a nightly suite that we use as a go/no-go test for releases.